src/lib/rules/manifest-app-name/manifest-app-name.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ * @fileoverview Check if the web app manifest file includes the `name`
+ * and `short_name` member with appropriate values.
+ */
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const { ucs2 } = require('punycode');
+
+import { debug as d } from '../../utils/debug';
+import { IManifestFetchEnd, IResponse, IRule, IRuleBuilder } from '../../types'; // eslint-disable-line no-unused-vars
```

Comment:
With the latest TS parsers you probably don't need the `//eslint-disable-line no-unused-vars`. I'm doing a PR that removes most of them.

Reply:
I'll remove it.

Reply:
> remove

Done.